Sources/KituraTemplateEngine/TemplateEngine.swift

```diff
@@ -21,6 +21,22 @@ public struct NullRenderingOptions: RenderingOptions {
     public init() {}
 }
 
+public enum TemplateEngineError: Swift.Error {
+    case needToImplementForUseWithoutKey
+    case unableToCastJSONToDict(Any)
```

Comment:
This seems to be unused?

Reply:
`unableToCastJSONToDict ` is used by KituraStencil and KituraMustache. 
I felt as it's used by both I should just add it here where it's available to both.